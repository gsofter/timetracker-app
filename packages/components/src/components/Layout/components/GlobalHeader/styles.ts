import { Property, Properties } from 'csstype';

export const styleSheet: { [key: string]: (obj) => Properties } = {
    globalHeaderStyle: ({ theme, primaryColor, layout }) => ({
        display: 'inherit',
        '& .ant-pro-global-header': {
            position: 'relative',
            display: 'flex',
            alignItems: 'center',
            height: '100%',
            padding: '0 16px',
            background: '#fff',
            boxShadow: '0 1px 4px rgba(0,21,41,0.08)',
        },
        '& .ant-pro-global-header > *': {
            height: '100%',
        },
        '& .ant-pro-global-header-collapsed-button': {
            display: 'flex',
            alignItems: 'center',
            marginLeft: '16px',
            fontSize: '20px',
        },
        '& .ant-pro-global-header-layout-mix': {
            backgroundColor: '#001529',
        },
        '& .ant-pro-global-header-layout-mix .ant-pro-global-header-logo h1': {
            color: '#fff',
        },
        '& .ant-pro-global-header-layout-mix .anticon': {
            color: '#fff',
        },
        '& .ant-pro-global-header-logo': {
            position: 'relative',
            overflow: 'hidden',
        },
        '& .ant-pro-global-header-logo a': {
            display: 'flex',
            alignItems: 'center',
            height: '100%',
        },
        '& .ant-pro-global-header-logo a img': {
            height: '28px',
        },
        '& .ant-pro-global-header-logo a h1': {
            height: '32px',
            margin: '0 0 0 12px',
            color: '#1890ff',
            fontWeight: '600',
            fontSize: '18px',
            lineHeight: '32px',
        },
        '& .ant-pro-global-header-menu .anticon': {
            marginRight: '8px',
        },
        '& .ant-pro-global-header-menu .ant-dropdown-menu-item': {
            minWidth: '160px',
        },
        '& .ant-pro-global-header .dark': {
            height: '48px',
        },
        '& .ant-pro-global-header .dark .action': {
            color: 'rgba(255,255,255,0.85)',
        },
        '& .ant-pro-global-header .dark .action > i': {
            color: 'rgba(255,255,255,0.85)',
        },
        '& .ant-pro-global-header .dark .action:hover, .ant-pro-global-header .dark .action.opened': {
            background: '#1890ff',
        },
        '& .ant-pro-global-header .dark .action .ant-badge': {
            color: 'rgba(255,255,255,0.85)',
        },

    }),
};

