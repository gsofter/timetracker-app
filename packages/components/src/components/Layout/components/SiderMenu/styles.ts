import { Property, Properties } from 'csstype'

export const styleSheet: { [key: string]: (obj) => Properties } = {
    siderMenuStyles: ({ theme, primaryColor }) => ({
        display: 'inherit',
        '& .ant-pro-sider': {
            position: 'relative',
            borderRight: '0',
            transition: 'background-color 0.3s',
        },
        '& .ant-layout-sider': {
            height: '100%',
        },
        '& .ant-pro-sider.ant-menu-vertical .ant-menu-item:not(:last-child), .ant-pro-sider.ant-menu-vertical-left .ant-menu-item:not(:last-child), .ant-pro-sider.ant-menu-vertical-right .ant-menu-item:not(:last-child), .ant-pro-sider.ant-menu-inline .ant-menu-item:not(:last-child)': {
            marginBottom: '4px',
        },
        '& .ant-drawer-close': {
            top: '-13px',
            right: '-17px',
            color: theme === 'light' ? primaryColor : '#fff',
        },
        '& .ant-drawer-content': {
            backgroundColor: theme === 'light' ? '#000' : '#fff',
        },
        '& .ant-pro-sider.ant-layout-sider-light .ant-menu-item a, .ant-menu-submenu-selected, .ant-menu-submenu-title:hover, .ant-menu-submenu-title i:hover ': {
            color: primaryColor ? primaryColor : '#1890ff',
        },
        '& .ant-menu-submenu-inline > .ant-menu-submenu-title:hover .ant-menu-submenu-arrow::before, .ant-menu-submenu-inline > .ant-menu-submenu-title:hover .ant-menu-submenu-arrow::after': {
            background: `linear-gradient(to right, ${primaryColor ? primaryColor : '#1890ff'}, ${primaryColor ? primaryColor : '#1890ff'})`,
        },
        '& .ant-pro-sider.ant-layout-sider-light, .ant-pro-sider.ant-layout-sider-light .ant-menu-item:not(.ant-menu-item-selected) a:hover, .ant-menu-item:hover': {
            color: primaryColor ? primaryColor : '#1890ff',
        },
        '& .ant-menu-item-selected a, .ant-menu-item-selected a:hover, .ant-menu:not(.ant-menu-horizontal) .ant-menu-item-selected a:hover': {
            color: '#fff',
        },
        '& .ant-menu:not(.ant-menu-horizontal) .ant-menu-item-selected': {
            background: primaryColor ? primaryColor : '#1890ff',
        },
        '& .removeBoxShadow ul': {
            boxShadow: 'none',
        },
        '& .ant-menu-inline .ant-menu-item::after': {
            borderRight: 'none',
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-item .anticon + span, .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-item .anticon + span, .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-submenu > .ant-menu-submenu-title .anticon + span, .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-submenu > .ant-menu-submenu-title .anticon + span': {
            display: 'none',
        },
        '& .ant-pro-sider ul.ant-menu-sub li.ant-menu-item .ant-pro-menu-item, .ant-pro-sider ul.ant-menu-sub li.ant-menu-submenu .ant-pro-menu-item': {
            paddingLeft: '8px',
        },
        '& .ant-pro-sider-logo': {
            position: 'relative',
            display: 'flex',
            alignItems: 'center',
            padding: '16px 16px',
            lineHeight: '32px',
            cursor: 'pointer',
        },
        '& .cu .ant-pro-sider-menu': {
            boxShadow: 'none',
        },
        '& .ant-pro-sider-logo > a': {
            display: 'flex',
            alignItems: 'center',
            justifyContent: 'center',
            minHeight: '32px',
        },
        '& .ant-pro-sider-logo img': {
            display: 'inline-block',
            height: '32px',
            verticalAlign: 'middle',
            transition: 'height 0.2s',
        },
        '& .ant-pro-sider-logo h1': {
            display: 'inline-block',
            height: '32px',
            margin: '0 0 0 12px',
            color: theme === 'light' ? primaryColor : '#fff',
            fontWeight: '600',
            fontSize: '18px',
            lineHeight: '32px',
            verticalAlign: 'middle',
            animation: 'fade-in',
            animationDuration: '0.2s',
        },
        '& .ant-pro-sider-extra': {
            marginBottom: '16px',
            padding: '0 16px',
        },
        '& .ant-pro-sider-extra-no-logo': {
            marginTop: '16px',
        },
        '& .ant-pro-sider-menu': {
            position: 'relative',
            zIndex: '10',
            minHeight: '100%',
            boxShadow: '2px 0 6px rgba(0,21,41,0.35)',
        },
        '& .ant-pro-sider .ant-layout-sider-children': {
            display: 'flex',
            flexDirection: 'column',
            height: '100%',
        },
        '& .ant-pro-sider .ant-layout-sider-children ::-webkit-scrollbar': {
            width: '6px',
            height: '6px',
        },
        '& .ant-pro-sider .ant-layout-sider-children ::-webkit-scrollbar-track': {
            background: 'rgba(255,255,255,0.15)',
            borderRadius: '3px',
            webkitBoxShadow: 'inset 0 0 5px rgba(37,37,37,0.05)',
        },
        '& .ant-pro-sider .ant-layout-sider-children ::-webkit-scrollbar-thumb': {
            background: 'rgba(255,255,255,0.2)',
            borderRadius: '3px',
            webkitBoxShadow: 'inset 0 0 5px rgba(255,255,255,0.05)',
        },
        '& .ant-pro-sider.ant-layout-sider-collapsed .ant-menu-inline-collapsed': {
            width: '48px',
        },
        '& .ant-pro-sider.ant-layout-sider-collapsed .ant-pro-sider-logo': {
            padding: '16px 8px',
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-item .sider-menu-item-img + span, .ant-pro-sider .-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-item .sider-menu-item-img + span, .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-submenu > .ant-menu-submenu-title .sider-menu-item-img + span': {
            display: 'inline-block',
            maxWidth: 0,
            opacity: 0,
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed .ant-menu-item': {
            width: '48px',
            padding: '0 16px !important',
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed .ant-pro-menu-item-title': {
            display: 'none',
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed .ant-menu-submenu-title': {
            width: '48px',
            padding: '0 16px !important',
        },
        '& .ant-pro-sider .ant-menu-item .anticon, .ant-pro-sider .ant-menu-submenu-title .anticon': {
            transition: 'none',
        },
        '& .ant-pro-sider-fixed': {
            position: 'fixed',
            top: '0',
            left: '0',
            zIndex: '99',
            height: '100%',
            overflow: 'auto',
            overflowx: 'hidden',
            boxShadow: '2px 0 8px 0 rgba(29,35,41,0.05)',
        },
        '& .ant-pro-sider-fixed > .ant-menu-root :not(.ant-pro-sider-link-menu)': {
            height: 'calc(100vh - 64px)',
            overflowY: 'auto',
        },
        '& .ant-pro-sider .ant-menu-inline .ant-menu-item, .ant-pro-sider .ant-menu-inline .ant-menu-submenu-title': {
            width: '100%',
        },
        '& .ant-pro-sider-light': {
            boxShadow: '2px 0 8px 0 rgba(29,35,41,0.05)',
        },
        '& .ant-pro-sider-light .ant-layout-sider-children ::-webkit-scrollbar-track': {
            background: 'rgba(0,0,0,0.06)',
            borderRadius: '3px',
            webkitBoxShadow: 'inset 0 0 5px rgba(0,21,41,0.05)',
        },
        '& .ant-pro-sider-light .ant-layout-sider-children ::-webkit-scrollbar-thumb': {
            background: 'rgba(0,0,0,0.12)',
            borderRadius: '3px',
            webkitBoxShadow: 'inset 0 0 5px rgba(0,21,41,0.05)',
        },
        '& .ant-pro-sider-light .ant-pro-sider-logo h1': {
            color: theme === 'light' ? primaryColor : '#fff',
        },
        '& .ant-pro-sider-light .ant-menu-light': {
            borderRightColor: 'transparent',
        },
        '& .ant-pro-sider-icon': {
            width: '14px',
            verticalAlign: 'baseline',
        },
        '& .ant-pro-sider-links': {
            width: '100%',
        },
        '&.ant-pro-sider-links ul.-menu-root': {
            height: 'auto',
        },
        '& .ant-pro-sider-collapsed-button': {
            borderTop: 'rgba(0,0,0,0.25)',
        },
        '& .ant-divider': {
            color: theme === 'light' ? 'rgba(0, 0, 0, 0.50)' : 'rgba(255, 255, 255, 0.65)',
            borderTopColor: theme === 'light' ? 'rgba(0, 0, 0, 0.50)' : 'rgba(255, 255, 255, 0.65)',
        },
        '& .ant-pro-sider-collapsed-button .anticon': {
            fontSize: '16px',
        },
        '& .ant-pro-sider .top-nav-menu li.ant-menu-item': {
            height: '100%',
            lineHeight: 1,
        },
        '& .ant-pro-sider .drawer .drawer-content': {
            //  background: ;
        },
        '& .ant-pro-sider .ant-menu-item .sider-menu-item-img + span, .ant-pro-sider .ant-menu-submenu-title .sider-menu-item-img + span': {
            opacity: 1,
            transition: 'opacity 0.3s , width 0.3s',
        },
        '& .ant-menu-inline-collapsed > .ant-menu-item, .ant-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-item, .ant-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-submenu > .ant-menu-submenu-title, .ant-menu-inline-collapsed > .ant-menu-submenu > .ant-menu-submenu-title': {
            padding: '0 24px',
        },
        '& .ant-pro-sider-menu.mobile-base-menu .ant-menu-submenu-title': {
            height: 'auto',
            lineHeight: '25px',
            whiteSpace: 'normal',
        },
        '& .ant-pro-sider-menu.mobile-base-menu .ant-menu-submenu-title .ant-pro-menu-item': {
            display: 'flex',
        },
        '& .ant-pro-sider-menu.mobile-base-menu .ant-menu-submenu-title .ant-pro-menu-item .anticon': {
            paddingTop: '5px',
        },
        '@keyframes fade-in': {
            '0%': {
                display: 'none',
                opacity: 0,
            },
            '99%': {
                display: 'none',
                opacity: 0,
            },
            '100%': {
                display: 'block',
                opacity: 1,
            },
        },
    }),
}
