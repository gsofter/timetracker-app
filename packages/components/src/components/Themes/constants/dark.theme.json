{
	"name": "dark",
	"type": "dark",
	"colors": {
		"primaryColor": "#1890ff",
		"app.background": "#fff",
		"app.headerHeight": "48px",
		"list.dropBackground": "#383b3d",
		"sideBarTitle.foreground": "#bbbbbb",
		"popover.background": "#1f1f1f",
		"popover.customize.border.color": "#3a3a3a",
		"body.background": "#000000",
		"component.background": "#141414",
		"layout.body.background": "#000000",
		"layout.header.background": "#1f1f1f",
		"layout.trigger.background": "#262626"
	},
	"tokenColors": [
		{
			"scope": "header",
			"settings": {
				"foreground": "#000080"
			}
		},
		{
			"scope": "text",
			"settings": {
				"foreground": "rgba(255,255,255,0.85)"
			}
		},
		{
			"scope": "text.secondary",
			"settings": {
				"foreground": "rgba(255,255,255,0.65)"
			}
		}
	]
}