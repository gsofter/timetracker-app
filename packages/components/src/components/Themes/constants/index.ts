export * from './actionTypes';
export { default as COLORS } from './colors';
export { default as THEMES } from './themes';
