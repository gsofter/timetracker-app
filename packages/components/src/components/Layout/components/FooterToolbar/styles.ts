import { IThemeState, getTokenColorsScopeSettings } from '../../../Themes/redux';
import { GlobalThemeSettings } from '../default-styles';


export const styleSheet = {
    footerToolbarStyle: ({ theme, prefixCls }: { theme: IThemeState, prefixCls: string }) => {
        const { antPrefix, componentBackground, borderColorSplit, shadow1Up } = GlobalThemeSettings;

        // less variables
        const proFooterBarPrefixCls = `${antPrefix}-pro-footer-bar`;


        return {
            [`& .${proFooterBarPrefixCls}`]: {
                position: 'fixed',
                right: '0',
                bottom: '0',
                'z-index': '99',
                display: 'flex',
                width: '100%',
                'align-items': 'center',
                padding: '0 24px',
                'line-height': '44px',
                background: `${componentBackground}`,
                'border-top': `1px solid ${borderColorSplit}`,
                'box-shadow': `${shadow1Up}`,
                '&-left': {
                    flex: '1',
                },

                '&-right': {
                    '> *': {
                        'margin-right': '8px',
                        '&:last-child': {
                            margin: '0',
                        }
                    }
                }
            },
        };
    },
}