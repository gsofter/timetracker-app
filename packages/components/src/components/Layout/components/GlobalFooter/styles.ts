// import { IThemeState, getTokenColorsScopeSettings } from '../../../Themes/redux';
import { ITheme } from '../../../Themes/interfaces';

import { GlobalThemeSettings } from '../default-styles';
import { getTokenColorsScopeSettings } from '../../../Themes/utils';


export const styleSheet = {
  globalFooter: ({ theme, prefixCls }: { theme: ITheme, prefixCls: string }) => {
    const { antPrefix, fontSizeBase } = GlobalThemeSettings;

    // less variables
    const proGlobalFooterPRefixCls = `${antPrefix}-pro-global-footer`;
    const textColorSecondary = getTokenColorsScopeSettings(theme, 'text.secondary').foreground;
    const textColor = getTokenColorsScopeSettings(theme, 'text').foreground;

    return {
      [`& .${proGlobalFooterPRefixCls}`]: {
        margin: '48px 0 24px 0',
        padding: '0 16px',
        'text-align': 'center',

        '&-links': {
          'margin-bottom': ' 8px',

          '& a': {
            color: `${textColorSecondary}`,
            transition: 'all 0.3s',

            '&:not(:last-child)': {
              'margin-right': '40px',
            },

            '&:hover': {
              color: `${textColor}`,
            },
          },
        },

        '&-copyright': {
          color: `${textColorSecondary}`,
          'font-size': `${fontSizeBase}`,
        },
      },
    };
  },
};
