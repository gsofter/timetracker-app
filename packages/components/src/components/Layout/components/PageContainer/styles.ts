import { IThemeSettings } from '../../../Themes/interfaces';
import { GlobalThemeSettings } from '../default-styles';


export const styleSheet = {
	pageContainer: ({ theme }: { theme: IThemeSettings }) => {
		const { antPrefix, screenLg, screenMd, screenSm, screenXl } = GlobalThemeSettings;

		// less variables
		const proLayoutPageContainer = `${antPrefix}-pro-page-container`;


		return {
			[`& .${proLayoutPageContainer}-children-content`]: {
				margin: '24px 24px 0',
			},
			[`& .${proLayoutPageContainer}`]: {

				'&-warp': {
					'background-color': `${GlobalThemeSettings.componentBackground}`,
					[`& .${antPrefix}-tabs-nav`]: {
						margin: '0',
					},
				},
				'&-ghost': {
					[`& .${proLayoutPageContainer}-warp`]: {
						'background-color': 'transparent',
					},

					[`& .${proLayoutPageContainer}-children-content`]: {
						'margin-top': '0',
					}
				}
			},
			[`& .${proLayoutPageContainer}-main`]: {
				[`& .${proLayoutPageContainer}-detail`]: {
					display: 'flex',
				},

				[`& .${proLayoutPageContainer}-row`]: {
					display: 'flex',
					width: '100%',
				},

				[`& .${proLayoutPageContainer}-title-content`]: {
					'margin-bottom': '16px',
				},

				[`& .${proLayoutPageContainer}-title`]: {
					flex: 'auto',

				},
				[`& .${proLayoutPageContainer}-content`]: { // merge it into one
					flex: 'auto',
					'margin-bottom': '16px',
				},

				[`& .${proLayoutPageContainer}-extraContent`]: { // merge it into one
					flex: '0 1 auto',
					'min-width': '242px',
					'margin-left': '88px',
					'text-align': 'right',
				},
				[`& .${proLayoutPageContainer}-main`]: {
					flex: '0 1 auto',

					width: '100%',
				},
				[`& .${proLayoutPageContainer}-logo`]: {
					'margin-bottom': '16px',
				},
			},
			[`@media screen and (max-width: ${screenXl})`]: {
				[`& .${proLayoutPageContainer}-main`]: {
					[`& .${proLayoutPageContainer}-extraContent`]: {
						'margin-left': '44px',
					}
				}
			},
			
			[`@media screen and (max-width: ${screenLg})`]: {
				[`& .${proLayoutPageContainer}-main`]: {
					[`& .${proLayoutPageContainer}-extraContent`]: {
						'margin-left': '20px',
					}
				}
			},
			
			[`@media screen and (max-width: ${screenMd})`]: {
				[`& .${proLayoutPageContainer}-main`]: {
					[`& .${proLayoutPageContainer}-row`]: {
						display: 'block',
					},
			
					[`& .${proLayoutPageContainer}-action`]: {
						'margin-left': '0',
						'text-align': 'left',
					},
					[`& .${proLayoutPageContainer}-extraContent`]: {
						'margin-left': '0',
						'text-align': 'left',
					}
				}
			},
			
			[`@media screen and (max-width: ${screenSm})`]: {
				[`& .${proLayoutPageContainer}-detail`]: {
					display: 'block',
				},
				[`& .${proLayoutPageContainer}-extraContent`]: {
					'margin-left': '0',
				},
			}
		};
	},
};
