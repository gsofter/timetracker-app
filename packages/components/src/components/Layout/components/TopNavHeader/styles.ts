import { getTokenColorsScopeSettings } from '../../../Themes/redux/utils';
import { IThemeState } from '../../../Themes/redux';
import { GlobalThemeSettings, LocalThemeSettings } from '../default-styles';


export const styleSheet = {
    topNavHeader: ({ theme, prefixCls }: { theme: IThemeState, prefixCls: string }) => {
        const { defaultSettings: { colors: { primaryColor }, type: themeName } } = theme;
        const { componentBackground, btnPrimaryColor, antPrefix } = GlobalThemeSettings;

        const headerColor = getTokenColorsScopeSettings(theme, 'header');
        // less variables
        const topNavHeaderPrefixCls = `${antPrefix}-pro-top-nav-header`;
        const headingColor = headerColor.foreground;
        return {
            [`& .${topNavHeaderPrefixCls}`]: {
                position: 'relative',
                width: '100%',
                height: '100%',
                'box-shadow': '0 1px 4px 0 rgba(0, 21, 41, 0.12)',
                transition: 'background 0.3s, width 0.2s',

                [`& .${antPrefix}-menu`]: {
                    background: 'transparent',
                },
                '&.light': {
                    'background-color': `${componentBackground}`,
                    [`& .${topNavHeaderPrefixCls}-logo`]: {
                        '& h1': {
                            color: `${headingColor}`,
                        },
                    },
                    '& .anticon': {
                        color: 'inherit',
                    }
                },
                '&-main': {
                    display: 'flex',
                    height: '100%',
                    'padding-left': '16px',
                    '&-left': {
                        display: 'flex',
                        'min-width': '192px',
                    },
                },

                '& .anticon': {
                    color: `${btnPrimaryColor}`,
                },
                '&-logo': {
                    position: 'relative',
                    'min-width': '165px',
                    height: '100%',
                    overflow: 'hidden',

                    '& img': {
                        display: 'inline-block',
                        height: '32px',
                        'vertical-align': 'middle',
                    },

                    '& h1': {
                        display: 'inline-block',
                        margin: '0 0 0 12px',
                        color: `${btnPrimaryColor}`,
                        'font-weight': '400',
                        'font-size': '16px',
                        'vertical-align': 'top',
                    }
                }
            },
        };
    },
};
