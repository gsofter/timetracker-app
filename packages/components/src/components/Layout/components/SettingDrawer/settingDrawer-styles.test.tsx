import 'raf/polyfill'
import * as React from 'react';
import { createSnapshot } from 'jest-react-fela';
import { createComponent } from 'react-fela';
import { THEMES } from '../../../Themes/constants/index';

import { styleSheet } from './styles';
import 'jest';


const rule = (props) => styleSheet.settingDrawer({
  theme: THEMES[0].defaultSettings
});
describe('Test styles', () => {

  it('should return a Component', () => {


    const Component = createComponent(rule)

    expect(createSnapshot(<Component className='ant-pro' />)).toMatchSnapshot()
  })

})


