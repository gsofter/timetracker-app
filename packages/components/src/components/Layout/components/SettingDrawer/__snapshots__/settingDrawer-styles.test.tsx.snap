// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test styles should return a Component 1`] = `
".a .ant-pro-setting-drawer-content {
  position: relative;
}
.b .ant-pro-setting-drawer-content {
  min-height: 100%;
}
.c .ant-pro-setting-drawer-content .ant-list-item span {
  flex: 1;
}
.d .ant-pro-setting-drawer-block-checkbox {
  display: flex;
}
.e .ant-pro-setting-drawer-block-checkbox-item {
  position: relative;
}
.f .ant-pro-setting-drawer-block-checkbox-item {
  margin-right: 16px;
}
.g .ant-pro-setting-drawer-block-checkbox-item {
  box-shadow: 0 1px 2.5px 0 rgba(0, 0, 0, 0.18);
}
.h .ant-pro-setting-drawer-block-checkbox-item {
  border-radius: 4px;
}
.i .ant-pro-setting-drawer-block-checkbox-item {
  overflow: hidden;
}
.j .ant-pro-setting-drawer-block-checkbox-item {
  cursor: pointer;
}
.k .ant-pro-setting-drawer-block-checkbox-item {
  width: 44px;
}
.l .ant-pro-setting-drawer-block-checkbox-item {
  height: 36px;
}
.m .ant-pro-setting-drawer-block-checkbox-item {
  background-color: #f0f2f5;
}
.n .ant-pro-setting-drawer-block-checkbox-item::before {
  position: absolute;
}
.o .ant-pro-setting-drawer-block-checkbox-item::before {
  top: 0;
}
.p .ant-pro-setting-drawer-block-checkbox-item::before {
  left: 0;
}
.q .ant-pro-setting-drawer-block-checkbox-item::before {
  width: 33%;
}
.r .ant-pro-setting-drawer-block-checkbox-item::before {
  height: 100%;
}
.s .ant-pro-setting-drawer-block-checkbox-item::before {
  background-color: #fff;
}
.t .ant-pro-setting-drawer-block-checkbox-item::before {
  content: \\"\\";
}
.u .ant-pro-setting-drawer-block-checkbox-item::after {
  position: absolute;
}
.v .ant-pro-setting-drawer-block-checkbox-item::after {
  top: 0;
}
.w .ant-pro-setting-drawer-block-checkbox-item::after {
  left: 0;
}
.x .ant-pro-setting-drawer-block-checkbox-item::after {
  width: 100%;
}
.y .ant-pro-setting-drawer-block-checkbox-item::after {
  height: 25%;
}
.z .ant-pro-setting-drawer-block-checkbox-item::after {
  background-color: #fff;
}
.ab .ant-pro-setting-drawer-block-checkbox-item::after {
  content: \\"\\";
}
.ac .ant-pro-setting-drawer-block-checkbox-item-light::before {
  background-color: #fff;
}
.ae .ant-pro-setting-drawer-block-checkbox-item-light::before {
  content: \\"\\";
}
.af .ant-pro-setting-drawer-block-checkbox-item-light::after {
  background-color: #fff;
}
.ag .ant-pro-setting-drawer-block-checkbox-item-dark::before {
  background-color: #1f1f1f;
}
.ah .ant-pro-setting-drawer-block-checkbox-item-dark::before {
  content: \\"\\";
}
.ai .ant-pro-setting-drawer-block-checkbox-item-dark::before {
  z-index: 1;
}
.aj .ant-pro-setting-drawer-block-checkbox-item-dark::after {
  background-color: #fff;
}
.ak .ant-pro-setting-drawer-block-checkbox-item-side::before {
  background-color: #1f1f1f;
}
.al .ant-pro-setting-drawer-block-checkbox-item-side::before {
  content: \\"\\";
}
.am .ant-pro-setting-drawer-block-checkbox-item-side::before {
  z-index: 1;
}
.an .ant-pro-setting-drawer-block-checkbox-item-side::after {
  background-color: #fff;
}
.ao .ant-pro-setting-drawer-block-checkbox-item-top::before {
  background-color: transparent;
}
.ap .ant-pro-setting-drawer-block-checkbox-item-top::before {
  content: \\"\\";
}
.aq .ant-pro-setting-drawer-block-checkbox-item-top::after {
  background-color: #1f1f1f;
}
.ar .ant-pro-setting-drawer-block-checkbox-item-mix::before {
  background-color: #fff;
}
.as .ant-pro-setting-drawer-block-checkbox-item-mix::before {
  content: \\"\\";
}
.at .ant-pro-setting-drawer-block-checkbox-item-mix::after {
  background-color: #1f1f1f;
}
.au .ant-pro-setting-drawer-block-checkbox-selectIcon {
  position: absolute;
}
.av .ant-pro-setting-drawer-block-checkbox-selectIcon {
  bottom: 4px;
}
.aw .ant-pro-setting-drawer-block-checkbox-selectIcon {
  right: 6px;
}
.ax .ant-pro-setting-drawer-block-checkbox-selectIcon {
  color: #1890ff;
}
.ay .ant-pro-setting-drawer-block-checkbox-selectIcon {
  font-weight: bold;
}
.az .ant-pro-setting-drawer-block-checkbox-selectIcon {
  font-size: 14px;
}
.ba .ant-pro-setting-drawer-block-checkbox-selectIcon {
  pointer-events: none;
}
.bb .ant-pro-setting-drawer-block-checkbox-selectIcon .action {
  color: #1890ff;
}
.bc .ant-pro-setting-drawer-color_block {
  display: inline-block;
}
.bd .ant-pro-setting-drawer-color_block {
  width: 38px;
}
.be .ant-pro-setting-drawer-color_block {
  height: 22px;
}
.bf .ant-pro-setting-drawer-color_block {
  margin: 4px;
}
.bg .ant-pro-setting-drawer-color_block {
  margin-right: 12px;
}
.bh .ant-pro-setting-drawer-color_block {
  vertical-align: middle;
}
.bi .ant-pro-setting-drawer-color_block {
  border-radius: 4px;
}
.bj .ant-pro-setting-drawer-color_block {
  cursor: pointer;
}
.bk .ant-pro-setting-drawer-title {
  margin-bottom: 12px;
}
.bl .ant-pro-setting-drawer-title {
  color: #262626;
}
.bm .ant-pro-setting-drawer-title {
  font-size: 14px;
}
.bn .ant-pro-setting-drawer-title {
  line-height: 22px;
}
.bo .ant-pro-setting-drawer-handle {
  position: absolute;
}
.bp .ant-pro-setting-drawer-handle {
  top: 240px;
}
.bq .ant-pro-setting-drawer-handle {
  right: 300px;
}
.br .ant-pro-setting-drawer-handle {
  z-index: 0;
}
.bs .ant-pro-setting-drawer-handle {
  display: flex;
}
.bt .ant-pro-setting-drawer-handle {
  align-items: center;
}
.bu .ant-pro-setting-drawer-handle {
  justify-content: center;
}
.bv .ant-pro-setting-drawer-handle {
  width: 48px;
}
.bw .ant-pro-setting-drawer-handle {
  height: 48px;
}
.bx .ant-pro-setting-drawer-handle {
  font-size: 16px;
}
.by .ant-pro-setting-drawer-handle {
  text-align: center;
}
.bz .ant-pro-setting-drawer-handle {
  background: #1890ff;
}
.ca .ant-pro-setting-drawer-handle {
  border-radius: 4px 0 0 4px;
}
.cb .ant-pro-setting-drawer-handle {
  cursor: pointer;
}
.cc .ant-pro-setting-drawer-handle {
  pointer-events: auto;
}
.cd .ant-pro-setting-drawer-production-hint {
  margin-top: 16px;
}
.ce .ant-pro-setting-drawer-production-hint {
  font-size: 12px;
}


<div className=ant-pro a b c d e f g h i j k l m n o p q r s t u v w x y z ab ac ae af ag ah ai aj ak al am an ao ap aq ar as at au av aw ax ay az ba bb bc bd be bf bg bh bi bj bk bl bm bn bo bp bq br bs bt bu bv bw bx by bz ca cb cc cd ce />;
"
`;
